src/components/UI/molecules/TagRow/TagRow.jsx

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-use-before-define */
```

Comment:
Good job lewis,

I guess you are disabling this rule so you could use the es5 function without defining before you use it in your component,
This means you are hoisting the function to the top,

I would like to know if there is a special reason, you are relying on `Hoisting` for the renderTags() method??

Reply:
Code clarity as advised by TTL as explained during the call